src/style.css

```diff
@@ -0,0 +1,129 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+.cards {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  max-width: 200px;
+  padding: 32px 16px 16px;
+}
+
+.cards__img {
+  margin-bottom: 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.cards__text {
+  margin-right: 0;
+  margin-bottom: 4px;
```

Comment:
You should create separate file for each BEM block

[checklist point 11](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)

detailed info how to do it in [this video](https://mate.academy/learn/css-advanced/bem#/video/917)